src/main/kotlin/com/demonwav/mcdev/i18n/lang/structure/I18nStructureViewModel.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n.lang.structure
+
+import com.intellij.ide.structureView.StructureViewModel
+import com.intellij.ide.structureView.StructureViewModelBase
+import com.intellij.ide.structureView.StructureViewTreeElement
+import com.intellij.ide.util.treeView.smartTree.Sorter
+import com.intellij.psi.PsiFile
+
+class I18nStructureViewModel(psiFile: PsiFile) : StructureViewModelBase(psiFile, I18nStructureViewElement(psiFile)), StructureViewModel.ElementInfoProvider {
+    override fun getSorters(): Array<Sorter> {
```

Comment:
You can use inline methods for all the 3 methods in this class to make it more concise